### DISCUSSION TITLE
Stimulus coding in various HSSM models

### DISCUSSION THREAD

**Discussion by hyang336** [lnccbrown/HSSM#475] [Q&A] [answered]
Dear HSSM community,

What is the correct way to use stimulus coding for responses and models? If using DDM, I take that "-1" means "incorrect" and "1" means "correct" in the response column. Could I instead use those two values to represent two response options instead of accuracy? For something like race model with more than two response options, is the response by-default stimulus coded (i.e., each particle represents evidence accumulation of one response option)?

Thanks!

**Comment by digicosmos86** [comment]
Hi @hyang336,

We had a discussion today and we are going to implement stim coding in the next few months. Please stay tuned :)

**Reply by tigretigre** [reply]
Any updates on this one? Anything I can do to contribute to move it along more quickly?

**Comment by frankmj** [comment, marked as answer]
Hi there,

In the interim you should be able to do stimulus coding by including a stim
column that tells the model which response should have been made for that
stimulus, and a response column that corresponds to which response was made
(rather than correct and errors). Then if you coded stim as simply 1 and -1
then specify v ~ stim this should give positive coefficients for stim.

Michael
